### DISCUSSION TITLE
Add cronexpression for 0 0 L * *  for the last day of a month

### DISCUSSION THREAD

**Discussion by Maurice-van-Beers** [atifaziz/NCrontab#123] [Ideas]
Hi Atifaziz, 
Is it possible to add support to specify the last day of the month: 0 0 L * *
Kind regards

**Comment by atifaziz** [comment]
See https://github.com/atifaziz/NCrontab/issues/9#issuecomment-205448445 on issue #9.

**Comment by springcomp** [comment]
Inspired by #127, [this can be built on top of NCrontab](https://gist.github.com/springcomp/949aed36b2a3392a3d76cee644288837).

- `0 0 L * *` last day of the month
- `0 0 * * 5L` last friday of the month
- `0 0 * * 1-5L` the last workdays of the month
- etc.
